### PR TITLE
fix(web): call postKeystroke on banner touch 🕊

### PR DESCRIPTION
**Pull request by mcdurdin**
Relates to #3621.

Merges into #5963.

Ensures that the `begin postKeystroke` entry point is called when the user accepts a suggestion.

@keymanapp-test-bot skip

We'll do testing separately later on this.

### CODE REVIEW

File: web/source/osk/banner.ts [hunk 1]
```diff
@@ -390,8 +390,28 @@ namespace com.keyman.osk {
       keyman.core.languageProcessor.addListener('suggestionsready', manager.updateSuggestions);
       keyman.core.languageProcessor.addListener('tryaccept', manager.tryAccept);
       keyman.core.languageProcessor.addListener('tryrevert', manager.tryRevert);
+      keyman.core.languageProcessor.addListener('suggestionapplied', this.suggestionApplied);
     }
 
+    /**
+     * Handler for post-porcessing once a suggestion has been applied: calls
```
Comment (rc-swag) [inline web/source/osk/banner.ts:397]:
```suggestion
     * Handler for post-processing once a suggestion has been applied: calls
```

**Review by rc-swag — COMMENTED** [review]
just noticed a typo. I think Darcy is probably best to give the final tick of approval.

**Review by darcywong00 — APPROVED** [review]
lgtm

File: web/source/osk/banner.ts (hunk 1 above)
Comment (mcdurdin) [reply web/source/osk/banner.ts:397]: I left this one there to see if anyone would pick it up.

Just kidding. Good catch 🐟

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.179-alpha](https://keyman.com/downloads/releases/alpha/15.0.179)

**Pull request merged**